tests/integration/api/v1/contents/get.test.js

```diff
@@ -1673,6 +1579,110 @@ describe('GET /api/v1/contents', () => {
           },
         ]);
       });
+
+      const runRootContentTest = async (url) => {
```

Comment:
@aprendendofelipe @Rafatcb eu acabei usando um teste para outro teste, veja

Reply:
Acho que entendi o que você fez, mas a forma usual de fazer isso é usando [`test.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout).

`test.each(table)(name, fn, timeout)`

Aí você fornece um array de arrays com os valores testados, e que também serão usados para a criação dos títulos de cada teste.

Você juntou os testes que dão sempre os mesmos resultados, mas passando as variáveis de `table` também para o `expect`, você pode unificar todos os testes que está criando nesse PR.

Reply:
@mthmcalixto chegou a testar a sugestão do Felipe aqui, de usar `test.each`? Fiz aqui um exemplo para ver se era possível continuar com bons nomes nos testes, e é sim:

```js
test.only.each([
  { params: 'with_children=false', title: 'get default root content only' },
  { params: 'with_children=true', title: 'get default root content only' },
])('$title with "$params"', async ({ params }) => {
  // ...

  const response = await fetch(`${orchestrator.webserverUrl}/api/v1/contents?${params}`);
  const responseBody = await response.json();

  // ...

  if (params.includes('with_children=true')) {
    // ...
  }

  expect(responseBody).toStrictEqual(rootReturn);
});
```

Na lista de testes fica:

```
Content type query params
  ✓ get default root content only with "with_children=false" (629 ms)
  ✓ get default root content only with "with_children=true" (615 ms)
 ...
```

Se passar mais parâmetros para a API:

```js
// Um objeto passado para o .each
{ params: 'with_children=false&with_root=false', title: 'get default root content only' }

// O resultado
✓ get default root content only with "with_children=false&with_root=false" (743 ms)
```

Acho que não dá para formatar melhor o título, mas isso pode deixar os testes mais enxutos e continua legível o que é esperado.

Inclusive, nesses exemplos acho que o `title` poderia ser mais descritivo. Ficou um pouco confuso o que é "default root content". Se for uma referência à estratégia, pode citá-la, por exemplo "get relevant root content only".

Reply:
@Rafatcb estou mexendo agora, realmente fica muito melhor, logo faço a atualização.